### PR TITLE
381 gv Documentation_PlotBrainAtlas

### PR DESCRIPTION
**Pull request by giovannivolpe**

### CODE REVIEW

File: braph2/atlas/PlotBrainSurf.m [hunk 1]
```diff
@@ -1,46 +1,54 @@
 classdef PlotBrainSurf < handle & matlab.mixin.Copyable
-    % PlotBrainSurf  Plot of a brain surface
+    % PlotBrainSurf Plotand and manage a brain surface
```
Comment (egolol) [inline braph2/atlas/PlotBrainSurf.m:2]: Plot

File: braph2/atlas/PlotBrainSurf.m [hunk 2]
```diff
@@ -1,46 +1,54 @@
...
-    % collection of surfaces. These brain surfaces are in '.nv' format and
-    % whithin the folder ./braph2/atlas/brainsurfs/.
+    % collection of surfaces. These brain surfaces are in nv format and
+    % in the folder ./braph2/atlas/brainsurfs/.
     % This class provides the common methods needed to manage the plot of 
-    % the surface, via a custom panel called Brain Surface Settings where
+    % the surface, via a custom panel called 'Brain Surface Settings' where
     % the user can change lighting, material, camlight, shadning, colormap,
-    % facecolor, brain color, face color, edgecolor and  background color. 
+    % facecolor, brain color, face color, edgecolor and background color.
```
Comment (egolol) [inline braph2/atlas/PlotBrainSurf.m:12]: edgecolor,

File: braph2/atlas/PlotBrainSurf.m (hunk 1 above)
Comment (giovannivolpe) [reply braph2/atlas/PlotBrainSurf.m:2]: done

File: braph2/atlas/PlotBrainSurf.m (hunk 2 above)
Comment (giovannivolpe) [reply braph2/atlas/PlotBrainSurf.m]: done

**Pull request merged**